apps/storybook/src/YAxisZoom.stories.tsx

```diff
@@ -0,0 +1,52 @@
+import { Pan, ResetZoomButton, VisCanvas, YAxisZoom } from '@h5web/lib';
```

Comment:
`YZoom.stories.tsx` renamed to `YAxisZoom` (same as `XZoom.stories.tsx` above but here, git didn't notice).